runtime/src/accounts_db.rs

```diff
@@ -10058,7 +10068,7 @@ pub mod tests {
         let account = AccountSharedData::new(1, some_data_len, &key);
         let ancestors = vec![(some_slot, 0)].into_iter().collect();
 
-        let accounts = &[(&key, &account)];
+        let accounts = &[(&key, &account)][..];
```

Comment:
I pulled down the code and played around with this; I was trying to figure out why this `[..]` explicit slice notation was needed, since I thought arrays auto converted to slices... I couldn't figure it out though. Do you know?

Reply:
[see the example here](https://docs.rs/rayon/1.0.3/rayon/slice/trait.ParallelSlice.html#examples-2)
`assert_eq!(chunks, vec![&[1, 2][..], &[3, 4], &[5]]);`
I think it makes it explicitly a slice. I think the issue is a fixed size array (or slice) vs a slice. Since this was test code, I decided it wasn't worth more noodling on my end atm.

Reply:
I think the issue in the test is that the array includes a size, which causes the types to not match with the trait impl.